server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tables/TableInformation.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.monitor.rest.tables;
+
+import org.apache.accumulo.core.master.thrift.TableInfo;
+
+/**
+ *
+ * Generates table information as a JSON object
+ *
+ * @since 2.0.0
+ *
+ */
+public class TableInformation {
+
+  // Variable names become JSON keys
+  public String tablename, tableId, tableState;
+
+  public int tablets, onlineTablets;
+  public long recs, recsInMemory;
+
+  public double ingest, ingestByteRate, query, queryByteRate;
+
+  public CompactionsList majorCompactions, minorCompactions, scans;
+
+  private int queuedMajorCompactions, runningMajorCompactions, queuedMinorCompactions, runningMinorCompactions, queuedScans, runningScans;
```

Comment:
Yup, currently working on a fix for these, will be pushing soon (in it there's other fixes from the rest of the suggestions from the PR (not all of them though))